EventFilter/DTRawToDigi/plugins/DTuROSRawToDigi.cc

```diff
@@ -438,21 +437,25 @@ int DTuROSRawToDigi::theDDU(int crate, int slot, int link, bool tenDDU) {
   
   int ddu = 772;
 
-  if (crate == 1368) {
+  if (crate == 1368) { ddu = 775; }
+
+  if (crate == 1369) {
     if (slot < 7) 
       ddu = 770;
     else 
       ddu = 771;
   }
 
-  if (crate == 1370) {
-    if (slot > 6) 
+  if (crate == 1370) { ddu = 772; }
+ 
+  if (crate == 1371) {
+    if (slot < 7) 
```

Comment:
This is just to check: it was ">6" and now is "<7" (i.e. ">" became "<"), please confirm that this is what you want

Reply:
Yes, it is on purpose: it allows to configure the final existing hardware in a more flexible way. Thanks for the heads-up!